drivers/spi/spi_andes_atcspi200.c

```diff
@@ -0,0 +1,629 @@
+/*
+ * Copyright (c) 2019 Andes Technology Corporation.
```

Comment:
s/2019/2022

Reply:
Thanks for reviewing.

I have fixed this incorrect date.